spring-integration-file/src/main/java/org/springframework/integration/file/tail/OSDelegatingFileTailingMessageProducer.java

```diff
@@ -71,13 +72,13 @@ public boolean isLongLived() {
 	protected void onInit() {
 		Assert.notNull(getFile(), "File cannot be null");
 		super.onInit();
-		this.command = "tail " + this.options + " " + this.getFile().getAbsolutePath();
 	}
 
 	@Override
 	protected void doStart() {
 		super.doStart();
 		destroyProcess();
+        this.command = "tail " + this.options + " " + this.getFile().getAbsolutePath();
```

Comment:
Sure.  Will update my project settings in case I contribute more in the future.  Thanks!